app/Config/Schema/i18n.php

```diff
@@ -45,17 +47,17 @@ public function before($event = array()) {
  * @param array $event Schema object properties
  * @return void
  */
-	public function after($event = array()) {
-	}
-
-	public $i18n = array(
-		'id' => array('type' => 'integer', 'null' => false, 'default' => null, 'length' => 10, 'key' => 'primary'),
-		'locale' => array('type' => 'string', 'null' => false, 'length' => 6, 'key' => 'index'),
-		'model' => array('type' => 'string', 'null' => false, 'key' => 'index'),
-		'foreign_key' => array('type' => 'integer', 'null' => false, 'length' => 10, 'key' => 'index'),
-		'field' => array('type' => 'string', 'null' => false, 'key' => 'index'),
-		'content' => array('type' => 'text', 'null' => true, 'default' => null),
-		'indexes' => array('PRIMARY' => array('column' => 'id', 'unique' => 1), 'locale' => array('column' => 'locale', 'unique' => 0), 'model' => array('column' => 'model', 'unique' => 0), 'row_id' => array('column' => 'foreign_key', 'unique' => 0), 'field' => array('column' => 'field', 'unique' => 0))
-	);
+    public function after($event = array())
+    {
+    }
 
+    public $i18n = array(
+        'id' => array('type' => 'integer', 'null' => false, 'default' => null, 'length' => 10, 'key' => 'primary'),
+        'locale' => array('type' => 'string', 'null' => false, 'length' => 6, 'key' => 'index'),
+        'model' => array('type' => 'string', 'null' => false, 'key' => 'index'),
+        'foreign_key' => array('type' => 'integer', 'null' => false, 'length' => 10, 'key' => 'index'),
+        'field' => array('type' => 'string', 'null' => false, 'key' => 'index'),
+        'content' => array('type' => 'text', 'null' => true, 'default' => null),
+        'indexes' => array('PRIMARY' => array('column' => 'id', 'unique' => 1), 'locale' => array('column' => 'locale', 'unique' => 0), 'model' => array('column' => 'model', 'unique' => 0), 'row_id' => array('column' => 'foreign_key', 'unique' => 0), 'field' => array('column' => 'field', 'unique' => 0))
```

Comment:
__[staging][code_sniffer]__
* Array key not aligned correctly; expected 20 spaces but found 8
* Array with multiple values cannot be declared on a single line